client/my-sites/plans/jetpack-plans/product-lightbox/constants.ts

```diff
@@ -35,5 +43,7 @@ export const PRODUCT_OPTIONS_HEADER: Record< string, string > = {
 	[ PLAN_JETPACK_SECURITY_T1_YEARLY ]: translate( 'Choose a storage option:' ),
 	[ PLAN_JETPACK_SECURITY_T2_YEARLY ]: translate( 'Choose a storage option:' ),
 	[ PRODUCT_JETPACK_SOCIAL_BASIC ]: translate( 'Select your plan:' ),
+	[ PRODUCT_JETPACK_SOCIAL_BASIC_MONTHLY ]: translate( 'Select your plan:' ),
 	[ PRODUCT_JETPACK_SOCIAL_ADVANCED ]: translate( 'Select your plan:' ),
+	[ PRODUCT_JETPACK_SOCIAL_ADVANCED_MONTHLY ]: translate( 'Select your plan:' ),
```

Comment:
We need the monthly product additions to display the monthly variants for the Jetpack Social product

Reply:
This repetition seems a bit odd to me, but it would need a larger refactor to change it.

Reply:
I agree it can be better, but yeah perhaps a part of a bigger refactor